### PR TITLE
Add watson_ins

### PR DESCRIPTION
**Pull request by natekc**
This PR adds support for the Watson Inertial Measurement System DMS-SGP02. It is a small wrapper on top of the serial interface on the driver.

For documentation, see [README](https://github.com/AutonomousVehicleLaboratory/watson_ins/blob/master/README.md)

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -16832,6 +16832,21 @@ repositories:
       url: https://github.com/warthog-cpr/warthog_simulator.git
       version: kinetic-devel
     status: maintained
+  watson_ins:
+    doc:
+      type: git
+      url: https://github.com/AutonomousVehicleLaboratory/watson_ins.git
```
Comment (clalancette) [inline kinetic/distribution.yaml]: The source here has a license specified in the package.xml (GPLv3), but no COPYING file at the top-level or headers on the source files.  Please see https://www.gnu.org/licenses/gpl-3.0.html on how to do that.

**Review by clalancette — DISMISSED** [review]
Needs license files added to the source repository.

**Comment by tfoote** [comment]
@natekc Please take a moment to resolve the missing license otherwise we'll close this out for no response.

**Comment by natekc** [comment]
Hi yes, I'll make the changes tonight.

**Comment by natekc** [comment]
License files have been added to source repo.

**Pull request merged**